typescript/infra/scripts/funding/fund-relayers-from-deployer.ts

```diff
@@ -137,10 +150,17 @@ async function main() {
       await relayerKey.fetch();
       await fundRelayer(chainConnection, relayerKey, desiredBalance);
     }
+    balances.push({
+      chain,
+      address: funderAddress,
+      balance: (await chainConnection.signer!.getBalance()).toNumber() / 1e18,
```

Comment:
The `toNumber()` will throw bc of an overflow for any balance > `Number.MAX_SAFE_INTEGER` (which is 9007199254740991, or `0.009007199254740991` ether). The easiest way is probably to do something like:
```
parseFloat(
  ethers.utils.formatEther(
    await chainConnection.signer!.getBalance()
  )
)
```

Reply:
You know, the worst part about this is I have definitely run into this before and didn't think about it. 🤦‍♀️